Deliveries.ipynb

```diff
@@ -1454,10 +1454,388 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
```

Comment:
I think this code is not required here because all of it has been moved to other files.

Reply:
Actually I still can't understand. I am not getting you.

Reply:
Ohk now I get it, yes Yes